experimental/auth-logic/simple_auth_logic_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package auth_logic
+
+import(
+  "testing"
+)
+
+func TestSimpleAuthLogic(t *testing.T) {
+  assert := func(name string, want, got bool) {
+		if want != got {
+      t.Fatalf("Query \"%v\" failed. want %t got %t", name, want, got)
```

Comment:
This is not performing a real escaping, e.g. it does not work if the string contain quotes. You can use `%q` to get properly escaped strings.

```suggestion
      t.Fatalf("Query %q failed. want %t got %t", name, want, got)
```